Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:22.04
 
 ARG ARCH=amd64
+ENV SYSTEMD_IGNORE_CHROOT=yes
```

Comment:
According to the [systemd docs](https://systemd.io/ENVIRONMENT/), the proper value to use is `1`, can you change it to 1 and also validate it works?

Reply:
Sure